components/esp_system/Kconfig

```diff
@@ -246,16 +251,26 @@ menu "ESP System Settings"
             non-blocking mode without using REPL. If you want to output in blocking mode with REPL or
             input through this secondary port, please change the primary config to this port
             in `Channel for console output` menu.
+
         config ESP_CONSOLE_SECONDARY_NONE
             bool "No secondary console"
+
         config ESP_CONSOLE_SECONDARY_USB_SERIAL_JTAG
-            bool "USB_SERIAL_JTAG PORT"
+            bool "USB Serial/JTAG Controller"
             depends on !ESP_CONSOLE_USB_SERIAL_JTAG
             help
                 This option supports output through USB_SERIAL_JTAG port when the UART0 port is not connected.
                 The output currently only supports non-blocking mode without using the console.
                 If you want to output in blocking mode with REPL or input through USB_SERIAL_JTAG port,
                 please change the primary config to ESP_CONSOLE_USB_SERIAL_JTAG above.
+
+        config ESP_CONSOLE_SECONDARY_UART_DEFAULT
+            bool "UART0 Default"
+            depends on ! ESP_CONSOLE_UART_DEFAULT && ! ESP_CONSOLE_UART_CUSTOM
```

Comment:
to keep code complexity down, I just implemented the "Default" UART.  But it may be worthwhile to implement custom UART as well. maybe another time?